omnibus/files/private-chef-cookbooks/private-chef/templates/default/elasticsearch_jvm_es7.opts.erb

```diff
@@ -0,0 +1,63 @@
+## JVM Configuration
```

Comment:
I have not verified these, but are they based on prior version opts? And have we looked to see if we need to change anything based on what's new in ES7?

Reply:
Yes, We are referring the latest ES7 JVM configurations - https://github.com/elastic/elasticsearch/blob/7.9/distribution/src/config/jvm.options